pom.xml

```diff
@@ -50,13 +50,20 @@
             <artifactId>configuration-as-code</artifactId>
             <version>${configuration-as-code.version}</version>
             <scope>test</scope>
-          </dependency>
-          <dependency>
+        </dependency>
+        <dependency>
             <groupId>io.jenkins.configuration-as-code</groupId>
             <artifactId>test-harness</artifactId>
             <version>${configuration-as-code.version}</version>
             <scope>test</scope>
-          </dependency>
+        </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>5.10.0</version>
```

Comment:
I think you can remove the <version> here. It's part of the bom

Reply:
Tried removing the mockito, but `package does not exist` arises.

>  It's part of the bom

Do we need to import the BOM or it will be imported automatically?

Reply:
The version tag. Not the dependency